model/task/task.go

```diff
@@ -1984,9 +1984,13 @@ func GetTasksByVersion(versionID, sortBy string, statuses []string, variant stri
 	tasks := []Task{}
 	err := db.FindAllQ(Collection, query, &tasks)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
+	}
+	count, err := Count(query)
```

Comment:
I would call this something like "total" so that it's not confused with the count of results

Reply:
true